superset/examples/helpers.py

```diff
@@ -25,13 +25,14 @@
 from superset import app, db
 from superset.connectors.connector_registry import ConnectorRegistry
 from superset.models import core as models
+from superset.models.dashboard import Dashboard
+from superset.models.slice import Slice
 
 BASE_URL = "https://github.com/apache-superset/examples-data/blob/master/"
 
 # Shortcuts
 DB = models.Database
-Slice = models.Slice
-Dash = models.Dashboard
+Dash = Dashboard
```

Comment:
These shortcuts seem less useful now - should we remove them?

Reply:
Good idea, it makes the PR a little more involved because a bunch of things import those models from this helpers file, but getting rid of that weird indirection really just makes it more worth IMO.